package.json

```diff
@@ -10,6 +10,7 @@
   "nyc": {
     "exclude": [
       "helpers/",
+      "middlewares/",
```

Comment:
Why did you exclude `middlewares/`?

And when did `helpers/` get excluded?? 😏😏

Reply:
The recent state of develop branch has helpers/ excluded. As for middlewares, I had to include it because it's presence was responsible for the -0.05 drop in test coverage.